README.md

```diff
@@ -6,6 +6,14 @@
 
 An events app to discover events happening around the world using the Open Event Platform
 
+Application is available at:
+
+Google Play Store [here](https://play.google.com/store/apps/details?id=com.eventyay.attendee) and
+F-Droid [Here](https://f-droid.org/en/packages/com.eventyay.attendee/).
```

Comment:
Looks good. Just change the above 2 lines to `  [Google Play Store](https://play.google.com/store/apps/details?id=com.eventyay.attendee) and
 [F-Droid](https://f-droid.org/en/packages/com.eventyay.attendee/)`